ethsigner/subcommands/src/test/java/tech/pegasys/ethsigner/subcommands/HashicorpSubCommandTest.java

```diff
@@ -121,14 +121,14 @@ public void commandLineWithTlsDisabledParsesIntoVariables() {
 
   @Test
   public void nonIntegerInputForPortShowsError() {
-    final String cmdLine = modifyField(validCommandLine(), "port", "noInteger");
+    final String cmdLine = CmdlineHelpers.modifyField(validCommandLine(), "port", "noInteger");
```

Comment:
You can avoid multiple changes in this file by static import.

Reply:
done - nicer